### ISSUE TITLE
02.23.2023 crash while creating backup

### ISSUE THREAD

**Issue by FezVrasta** [J-D-K/JKSV#206]
Hi, I just modified a "new" Switch, loaded a vanilla Atmosphere 1.5.4 and installed JKSV 02.23.2023.

For some reason anytime I try to create a backup (Device - Animal Crossing New Horizons) or recover an existing one it crashes with the message below. I'm executing JKSV from the hbmenu from the Album app. It works if I launch it pressing R while launching a game.

On all my other consoles it works flawlessly, do you have any idea of what could be going on? Thanks!

![IMG_1543](https://github.com/J-D-K/JKSV/assets/5382443/89b02d7e-ddc2-4de8-8ddc-bac456d11aab)

**Comment by FezVrasta** [comment]
Hi, as I specified in the issue this only happens when I launch it from the Album. If I execute it while launching a game and pressing R it works.

**Comment by impeeza** [comment]
exactly, did you read the instructions on README.MD,  the app is only supported on Title replacement mode, because applet mode have so limited memory.

**Comment by FezVrasta** [comment]
I don't see that in the readme, it only mentions that in the "Misc Extras" section in relation to Process RomFS.

Also, this doesn't explain why JKSV works properly on all the other switches I tested and it crashes only on this one.

**Comment by idcha** [comment]
acc to #149 the applet mode may be disabled anyway in the future. (maybe i'm lost in translation?) I can reproduce crashes with MH Stories 2 in applet mode. Save files are 4x 8225kB each but the crash is on the first file which is left as 0 byte file.
